src/ptrace.rs

```diff
@@ -30,6 +32,26 @@ fn setregs(pid: Pid, regs: &Regs) -> nix::Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Get user registers, as with `ptrace(PTRACE_GETREGS, ...)`
+#[cfg(target_arch = "aarch64")]
+fn getregs(pid: Pid) -> nix::Result<Regs> {
+    ptrace_get_data::<Regs>(Request::PTRACE_GETREGSET, pid)
```

Comment:
The arguments for PTRACE_GETREGSET is a bit different. See here [ptrace(2)](https://man7.org/linux/man-pages/man2/ptrace.2.html). I think we can switch for both x86/aarch64 to this instead.